packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -930,6 +936,41 @@ contract Migration is Script, UsingRegistry, MigrationsConstants {
     );
   }
 
+  function migrateEpochManagerEnabler() public {
```

Comment:
This should be added to the L2 scripts rather than this one, right?

Reply:
not sure I understand